cmd/statusd-cli/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+)
+
+var (
+	statusdConn = flag.String("statusd", "localhost", "set host of statusd")
```

Comment:
Does it require `statusd` up and running? How does it know about a port number?

Reply:
Yes, `statusd` runs as extra daemon the CLI is connecting too. First draft had also port number configurable, now a standard port number is used. But will switch back to make it a parameter.